### DISCUSSION TITLE
開発への質問

### DISCUSSION THREAD

**Discussion by RL-Nadir** [sikabane-works/bakabakaband#422] [Q&A]
ネタとして仕込まれた可能性があると思い、こちらで伺います。
「床が行く手を阻んでいる」と表示されて、移動・採掘不可能な「床」地形が発生しました。
スクショの＠の上1マスに発生しています。
これって、そういうネタでしょうか。もしバグでしたら、issueの方にセーブデータも上げるようにします。

![image](https://user-images.githubusercontent.com/79453028/119503364-da05d180-bda5-11eb-8f92-197729dbee1e.png)

**Comment by deskull-m** [comment]
バグですね。トラップなどの隠蔽に用いている、表面上のIDと実際のIDを別にする仕様で実際のIDが何か不正な値をとっているときに発生します。前の新トラップ追加時になんかやらかした可能性が高いのでIssue化します。

**Reply by RL-Nadir** [reply]
ご対応お疲れさまです。
本件はバグとのことでしたが、実害のない形であれば、ギミックとして面白いかもな、とは思いました。
つまり、　'.' だまし絵（床）の壁　といった感じです。振る舞いは壁ですが、表示は床を表すドットです。
アリーナや、NO_ROOMタイプなどの、厚さ1マスの壁に、時々紛れ込んでいたり、アビス風ダンジョンには普通に出てきたりすると、ゲームとしてはともかく、面白いかも知れません。

以上、本件の感想文でした。

**Reply by deskull-m** [reply]
そうですね、偽装地形自体は別途Issue化してみたいと思います。
